### PR TITLE
Gwh apidocs 1846 publisher options page

### PR DESCRIPTION
**Pull request by genwhittTTD**
new Web Integration Overview for publishers.

### CODE REVIEW

File: docs/guides/integration-options-publisher-web.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+- You can keep your [directly identifying information (DII)](../ref-info/glossary-uid.md#gl-dii) entirely on the server side.
+- If your development resources are back-end developers, you might prefer a server-side integration.
+- For server-side Prebid integration, there is no requirement to update to the latest Prebid version. If you're using Prebid, and have a constraint with regard to version, choose server-side integration.
+
+The following table summarizes the options for publishers who want to generate the UID2 token on the server side.
+
+| Option | Documentation |
+| :--- | :--- |
+| UID2 SDK for JavaScript, server-side implementation | [JavaScript Standard Integration Guide](integration-javascript-standard.md) |
```
Comment (sunnywu) [inline docs/guides/integration-options-publisher-web.md:100]: why is Prebid.js server integration not listed here?

File: docs/guides/integration-options-publisher-web.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+| Prebid.js client-side implementation (8.21.0 or later) | [UID2 Client-Side Integration Guide for Prebid.js](integration-prebid-client-side.md) |
+| UID2 SDK for JavaScript, client-side implementation | [JavaScript Express Integration Guide](publisher-client-side.md) |
+
+### Server-Side Integration Options
+
+Generating the UID2 token on the server side has the following advantages:
+
+- You can keep your [directly identifying information (DII)](../ref-info/glossary-uid.md#gl-dii) entirely on the server side.
+- If your development resources are back-end developers, you might prefer a server-side integration.
+- For server-side Prebid integration, there is no requirement to update to the latest Prebid version. If you're using Prebid, and have a constraint with regard to version, choose server-side integration.
```
Comment (sunnywu) [inline docs/guides/integration-options-publisher-web.md:94]: the way i read this is "if i do server side integration with prebid.js, i can use any version of prebid.js", which isn't true and we specify 7.53.0 or later in other sections. 

just raising this and up to you if you wanna rephrase it.

Comment (genwhittTTD) [reply docs/guides/integration-options-publisher-web.md:94]: @sunnywu need to discuss to clarify.

Comment (genwhittTTD) [reply docs/guides/integration-options-publisher-web.md:94]: Good point, thx. Changed it to:
For server-side Prebid integration, there is no requirement to update to the latest Prebid version, as long as your version is 7.53.0 or later. If you're using Prebid, and have a constraint with regard to version, choose server-side integration.

File: docs/guides/integration-options-publisher-web.md (hunk 1 above)
Comment (sunnywu) [reply docs/guides/integration-options-publisher-web.md:119]: @genwhittTTD yes ignore this please. i read it wrong.

**Pull request merged**